Python/Product/TestAdapter.Executor/UnitTest/UnitTestTestDiscoverer.cs

```diff
@@ -138,16 +138,22 @@ ITestCaseDiscoverySink discoverySink
 
         public string[] GetArguments(IEnumerable<string> sources, PythonProjectSettings settings, string outputfilename) {
             var arguments = new List<string>();
-            arguments.Add(DiscoveryAdapterPath);
-            arguments.Add("discover");
-            arguments.Add("unittest");
-            arguments.Add("--output-file");
-            arguments.Add(outputfilename);
-            //Note unittest specific options go after this separator
-            arguments.Add("--");
-            arguments.Add(settings.UnitTestRootDir);
-            arguments.Add(settings.UnitTestPattern);
-
+            foreach (var source in sources.ToArray()) {
```

Comment:
if we are calling 'unitTest' for every source file. shouldn't we pass in the source file?

Reply:
@bschnurr yes you are right, there's only 1 element in the `sources.ToArray()`, so instead of a for loop, maybe I should just change to `sources.ToArray()[0]`.